shared/AppInsightsCore/Tests/Unit/src/CookieManager.Tests.ts

```diff
@@ -214,50 +214,93 @@ export class CookieManagerTests extends AITestClass {
         this.testCase({
             name: "CookieManager: Set as Disabled",
             test: () => {
+                let getCookieCallCount = 0;
+                let setCookieCallCount = 0;
+                let delCookieCallCount = 0;
+                
                 let manager = createCookieMgr({
                     cookieCfg: {
                         enabled: false,
-                        getCookie: () => { throw "Should not be called" },
-                        setCookie: () => { throw "Should not be called" },
-                        delCookie: () => { throw "Should not be called" }
+                        getCookie: () => { 
```

Comment:
Any tests which is only expecting 0 number of calls and never enables the cookie manager should continue to use the previous pattern -- revert this change in this test